docs/man_pages/bodhi.rst

```diff
@@ -41,6 +41,11 @@ Most of the commands will accept these options:
 
     Many commands accept this flag to specify which user's updates should be operated upon.
 
+``--mine``
+
+    Some commands accept this flag instead of the ``--user <username>`` option. This uses the
+    value from the ``USERNAME`` environment variable.
```

Comment:
As is, only one command uses it. However, it does seem natural to add this to ```bodhi updates query --mine```. Want to add that as a flag in the code too, so this docs statement is accurate about "some commands"? ☺